config/credentials.yaml

```diff
@@ -17,3 +17,9 @@ credentials:
         domain:
           description: "GitHub API domain"
           name: "api.github.com"
+      - credentials:
+          - string:
+              scope: GLOBAL
+              id: slack-token
+              secret: '${SLACK_TOKEN}'
+              description: Slack token
```

Comment:
Could sort alphabetically and be consistent about single/double/no quotes regarding the other credentials.
If we choose to never use them, why not, then please add a first commit to clean up the file.